_pytest/runner.py

```diff
@@ -494,9 +494,12 @@ def importorskip(modname, minversion=None):
     """
     __tracebackhide__ = True
     compile(modname, '', 'eval') # to catch syntaxerrors
+    should_skip = False
     try:
         __import__(modname)
     except ImportError:
+        should_skip = True
+    if should_skip:
```

Comment:
A chained exception with __context__ set was raised there. Prior to the changes it was ignored, but when I added processing of exception chain it caused a test failure. IIRC the test failed because the trace back of original exception (ImportError) was filtered and no traceback entries were in the entry list.